client/src/pages/Search/AddressSearch.js

```diff
@@ -22,15 +26,25 @@ export default function AddressSearch({ updateFields }) {
       componentRestrictions: { country: "usa" },
     },
   });
-  
+
+  function handleUserAddressError() {
+    dispatch(
+      alertSnackbar({
+        message: "Please Enable Location Services",
```

Comment:
nit: write this like a sentence instead of a title
"Please enable location services"